app/models/actor.rb

```diff
@@ -0,0 +1,3 @@
+class Actor < ApplicationRecord
+  has_and_belongs_to_many :movies
```

Comment:
This association doesn't always work with the many to many relationships we use in Mod 2 so we discourage folks from using it. Instead, use 
```
has_many :actor_movies
has_many :movies, through: :actor_movies
```
We don't like using has_and_belongs_to_many because it suggests that your join table can *only* have foreign keys and nothing else. There are times in our projects that your joins tables also have other attributes so we encourage learning this syntax instead! In this case, it's still going to work because we don't have attributes besides foreign keys in this joins table, but on projects, it might cause problems. This doesn't penalize you, just want you to be aware.